feeds/feed_test.go

```diff
@@ -0,0 +1,72 @@
+package feeds
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const schema_path = "../package.schema.json"
+
+type ExtendPackage struct {
```

Comment:
This can be `extendPackage` as it can remain private in scope.

Reply:
True, thanks!